engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -44,11 +44,16 @@ import lenthall.config.ScalaConfig.EnhancedScalaConfig
   lazy val jobStore: JobStore = new SqlJobStore(CromwellDatabase.databaseInterface)
   lazy val jobStoreActor = context.actorOf(JobStoreActor.props(jobStore), "JobStoreActor")
 
+  lazy val callCache: CallCache = new CallCache(CromwellDatabase.databaseInterface)
+  lazy val callCacheReadActor = context.actorOf(RoundRobinPool(25)
```

Comment:
This is saying that there'll always be a pool of 25 CallCacheReadActors alive. When a cache read request comes in, which one of the 25 gets the message is based on round-robin assignment